public/about.html

```diff
@@ -33,43 +33,260 @@
+          <a href="https://facebook.com/BantayBayanTayo" target="_blank"><img src="../assets/images/about_fb.png" class="abt_fb_1"></a>
+          <a href="https://l.facebook.com/l.php?u=http%3A%2F%2Ftwitter.com%2FBantayBayanTayo%3Ffbclid%3DIwAR1XGVB-ntc44bMnby-eIy2M2PxsyXfxhafhCHt-gfrmQFuWewhdFv_Bzi8&h=AT2VaHz6xjRRLGQ4U1AhXaaHJ1joIIB_UgjrqD2nqNQj33cuhgoLPO34STyFcbMJFNg8pBQquCteBqRsFzSUqBegPJ18J-jDkgWkZsokoHSfS-1W8ELTeOxdE55cVKHi7qmbi2_IKpInrIvGfRfeSg" target="_blank"><img src="../assets/images/about_twitter.png" class="abt_twitter_1"></a>
+        </div>
+        <p>
+          Ang GoodGov PH ay isang kilusan ng mga kabataang nagsusulong ng mabuting pamamahala 
+          (“Good Governance”) sa Pilipinas. Gumagawa ito ng mga programang naka-angkla sa advocacy 
+          campaigns, alliance-building, governance education, community collaboration, at innovation. 
+        </p>
+        <p>
+          Ito ay organisasyong pangkabataan na non-profit at non-partisan, at nakarehistro sa National
+          Youth Commission - Youth Organization Registration Program (YORP). 
+        </p>
+      </div>
+      <div class="partner_subcontent2">
+        <div class="partner_subcontent2_head">
+          <img src="../assets/images/dsc_loyola_logo.png" class="abt_dsc_logo">
+          <h4>DSC - Loyola</h4>
+          <a href="https://www.facebook.com/dscloyola/" target="_blank"><img src="../assets/images/about_fb.png" class="abt_fb_2"></a>
+          <a href="https://twitter.com/dscloyola/" target="_blank"><img src="../assets/images/about_twitter.png" class="abt_twitter_2"></a>
+        </div>
+        <p>
+          Developer Student Clubs Loyola is a student organization powered by Google Developers that aims 
+          to build students’ skills and network by giving them access to different technologies, 
+          specifically Google Developer technologies like Android, Firebase, Angular, Flutter, Google Cloud
+          Platform and many more. 
+        </p>
+        <p>
+          Together, we learn in a peer-to-peer learning environment and build solutions for the community.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="organizations">
+    <div class="organizations_label">
+      <p>Partner organizations</p>
+    </div>
+    <div class="org_content">
+      <div class="org1">
+        <img src="../assets/images/The Initiative PH 1.png" class="the_initiative_ph_logo">
+        <h4>The Initiative PH</h4>
+        <div>
+          <a href="#" target="_blank"><img src="../assets/images/about_fb.png" class="org_fb_2"></a>
+          <a href="#" target="_blank"><img src="../assets/images/about_twitter.png" class="org_twitter_2"></a>
+        </div>
+      </div>
+      <div class="org2">
+        <img src="../assets/images/Kadiwa 1.png" class="kadiwa_logo">
+        <h4>Kadiwa Student Coaliton</h4>
+        <div>
+          <a href="#" target="_blank"><img src="../assets/images/about_fb.png" class="org_fb_2"></a>
```

Comment:
rel="noopener noreferrer" should be added to links containing target="_blank" as a precaution against reverse tabnabbing. For more information, please refer to the following article:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
ask for kadiwa's social media links